src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj

```diff
@@ -8,22 +8,29 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" >
+    <ProjectReference Include="..\..\tools-local\Microsoft.ML.AutoML.SourceGenerator\Microsoft.ML.AutoML.SourceGenerator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj">
       <PrivateAssets>all</PrivateAssets>
     </ProjectReference>
     <ProjectReference Include="..\Microsoft.ML.CpuMath\Microsoft.ML.CpuMath.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.SearchSpace\Microsoft.ML.SearchSpace.csproj" />
```

Comment:
You will probably have to make this `PrivateAssets` since we aren't shipping a SearchSpace package. Actually, if AutoML is going to use the SearchSpace stuff, we either do need to ship it or have it be packaged with AutoML itself.

Reply:
Sure

Reply:
## Resolved

Reply:
We still need to either package SearchSpace or include its files in the automl nuget. Otherwise the nugets will have issues at runtime.

Reply:
I follow the solution used in `Microsoft.ML` which packs `Microsoft.ML.Core` together in nuget and mark `SearchSpace` with `IncludeInNuget`

Reply:
### Resolved